### PR TITLE
Platform specific version invoked by generic show.version

### PR DESCRIPTION
**Pull request by VenkatCisco**
#### What I did
This generic enhancement can invoke a platform specific show version method "if available" else skip. The routine is helpful to provide details of Vendor Specific SDK info or SKU etc. 

#### How I did it

Simply Invoke a  python platform routine for capturing any platform specific variables if exists. 

#### How to verify it

cisco@sonic:~$ show version

<...snip...>

Cisco Silicon One SDK: 1.33.1-SDK-1.5.2-17
Cisco BSP: 0.1.11-32-g19b3d77
Cisco FPD: 0.1-22-gfbde9e1

<...snip...>

### CODE REVIEW

File: show/main.py [hunk 1]
```diff
@@ -977,6 +977,12 @@ def version(verbose):
     click.echo("ASIC: {}".format(asic_type))
     click.echo("ASIC Count: {}".format(asic_count))
     click.echo("Serial Number: {}".format(serial_number.stdout.read().strip()))
+    try:
+        sonic_platform.cli.show.version(verbose)
```
Comment (arlakshm) [inline show/main.py:981]: is the `sonic_platform` module part of sonic-utilities repo? 
If other platforms also need to be dump platform specific information also how that can be achieved

**Review by arlakshm — COMMENTED** [review]
Can you add unit tests for this change?

File: show/main.py (hunk 1 above)
Comment (VenkatCisco) [reply show/main.py:981]: as per review comments, the code is restructured to the below: 

1) the underlying show_platform/cli/show.py will be moved under the respective plugin path as follows: 

          sonic-utilities/show/plugins/cisco-8000.py 

Similar future enhancements for config and clear options will be as: 

          sonic-utilities/config/plugins/cisco-8000.py
          sonic-utilities/clear/plugins/cisco-8000.py

**Comment by anshuv-mfst** [comment]
@arlakshm - can you please review and approve asap, thanks.

**Comment by arlakshm** [comment]
@VenkatCisco  as per our discussion the conclusion was to make changes to platform common module to provide the platform version information and display here

https://github.com/Azure/sonic-platform-common/tree/master/sonic_platform_base
Refer: chassis_base.py

**Comment by VenkatCisco** [comment]
This will be consumed thru PR#1578

**Pull request closed without merging**